### PR TITLE
TypeName to lowerCase

### PR DESCRIPTION
**Pull request by aurislt7**
Addon PR: https://bentleycs.visualstudio.com/iModelTechnologies/_git/imodel02/pullrequest/147521

### CODE REVIEW

**Review by grigasp — CHANGES_REQUESTED** [review]
Please:
- add a link to addon PR in description
- rush change
- extract-api

File: common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/presentation-common",
+      "comment": "Added a property `extendedType` in `PropertyType` interface.",
```
Comment (grigasp) [inline common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json:5]:
```suggestion
      "comment": "Added an [PropertyInfo.extendedType]($presentation-common) attribute.",
```

File: common/changes/@bentley/presentation-components/presentation-typeName-lowercase_2021-03-02-13-38.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/presentation-components",
+      "comment": "Updated `createPropertyDescription` in `ContentBuilder` to set property's typename to lowercase.",
```
Comment (grigasp) [inline common/changes/@bentley/presentation-components/presentation-typeName-lowercase_2021-03-02-13-38.json:5]: `ContentBuilder` is internal, so we shouldn't mention it here.
```suggestion
      "comment": "Always set [PropertyDescription.typename]($ui-abstract) to lowercase when creating content.",
```

File: common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json (hunk 1 above)
Comment (miniant-git) [reply common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json:5]: Is this really necessary? These links won't work outside our documentation, for example, [here](https://github.com/imodeljs/imodeljs/blob/master/presentation/common/CHANGELOG.md).

Comment (roluk) [reply common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json:5]: Is this really necessary? These links won't work outside our documentation, for example, [here](https://github.com/imodeljs/imodeljs/blob/master/presentation/common/CHANGELOG.md).

Comment (grigasp) [reply common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json:5]: Good point, thanks. Updated the suggestions.

**Pull request merged**